ios/RCTMarkdownUtils.mm

```diff
@@ -47,13 +48,17 @@ - (NSAttributedString *)parseMarkdown:(nullable NSAttributedString *)input
   JSValue *result = [function callWithArguments:@[inputString]];
   NSArray *ranges = [result toArray];
 
-  NSMutableAttributedString *attributedString = [[NSMutableAttributedString alloc] initWithString:inputString attributes:_backedTextInputView.defaultTextAttributes];
+  NSMutableAttributedString *attributedString = [input mutableCopy];
```

Comment:
I had to use `mutableCopy` here because creating `NSMutableAttributedString` from `NSString` loses the information about diacritics context.